### PR TITLE
Ol8 v1r5 small updates - update policy text & remove rule for OL08-00-010510

### PR DESCRIPTION
**Pull request by Xeicker**
#### Description:

- Update policy in rule `ssh_host_key_permissions`
- Remove rule `sshd_disable_compression` from OL8 & RHEL8

#### Rationale:

- This is to align with latest DISA STIG requirements. v1r5 for OL8 and v1r9 for RHEL8

### CODE REVIEW

File: products/rhel8/profiles/stig.profile [hunk 1]
```diff
@@ -31,7 +31,6 @@ selections:
     - var_accounts_user_umask=077
     - var_password_pam_difok=8
     - var_password_pam_maxrepeat=3
-    - var_sshd_disable_compression=no
```
Comment (Mab879) [inline products/rhel8/profiles/stig.profile:34]: This also needs to be removed from the profile stability.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR. Make sure that you update the variable in the RHEL 8 profile stability as well.

**Review by Mab879 — APPROVED** [review]
Thanks for the PR!

**Comment by vojtapolasek** [comment]
@ComplianceAsCode/oracle-maintainers  Could you please accept these changes?

**Review by freddieRv — APPROVED** [review]

**Pull request merged**